.ghci.semantic

```diff
@@ -1,61 +1,17 @@
 -- GHCI settings for semantic, collected by running cabal repl -v and checking out the flags cabal passes to ghc.
```

Comment:
All of the deletions in this file are now being handled via `script/ghci-flags` instead.